### PR TITLE
XW-1580 Pull translations

### PR DESCRIPTION
**Pull request by kamilkoterba**
Updating translations and update message keys to match API convention.
https://wikia-inc.atlassian.net/browse/XW-1580
@Wikia/x-wing

### CODE REVIEW

File: crowdin.conf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [crowdin.project]
 projectid = "design-system"
 repo.branch = "master"
-source.base = "."
+source.base = "./i18n"
```
Comment (Grunny) [inline crowdin.conf]: Can just be `i18n`, it will be made relative to the crowdin.conf file location anyway.

**Comment by Grunny** [comment]
LGTM (other than deciding what to do with `__community__`). :)

**Comment by rogatty** [comment]
LGTM

**Comment by kamilkoterba** [comment]
https://wikia-inc.atlassian.net/browse/XW-1740

**Pull request merged**